backend/src/database/database-factory.ts

```diff
@@ -0,0 +1,8 @@
+import { Database } from './database';
+import { PgDatabase } from './postgres/PostgresDatabase';
+
+export class DatabaseFactory {
+  static getDatabase(): Database {
+    return PgDatabase.instance();
+  }
+}
```

Comment:
Db factory that actually determines which instance of the db we are using (postgres or mongo)